### ISSUE TITLE
Waterfox Browser no longer cleaned

### ISSUE THREAD

**Issue by randyrandersn** [bleachbit/bleachbit#233]
Waterfox used to shared its profile directory with Firefox, so used to be able to clean up Waterfox usage data and logs using the Firefox entries in Bleachbit

Waterfox 55 moved all profile data from .mozilla to .waterfox so this no longer works.

Possible to add a section in Bleachbit exclusive to cleaning up Waterfox?

**Comment by ROCKNROLLKID** [comment]
I believe the directories are the same, but as you said, it is .waterfox instead of .mozilla. Should be able to modify the existing firefox entry for Waterfox.

**Comment by randyrandersn** [comment]
Thanks.

Waterfox is keeping legacy addon support when Firefox disables this is version 57. I think going forward there will be additional non trivial differences in .waterfox and .mozilla profiles and preferences.  

Just something to consider to ensure Waterfox data is adequately "cleaned" going forward.

In the meantime I have added the following files and directories to the custom locations in Preferences, and selected them via System -> Custom option.

```
~/.waterfox/[profile]/cookies.sqlite
~/.waterfox/[profile]/content-prefs.sqlite
~/.waterfox/[profile]/webappstore.sqlite
~/.cache/waterfox/[profile]
```
I also added ```~/.waterfox/[profile]/places.sqlite``` to clean up history, but it also deleted toolbars, bookmarks and menu entries too so I removed this.

Is there a way to vacuum custom .sqlite files too?  Be a good feature... :) .
